pkg/controller/controller.go

```diff
@@ -532,6 +532,26 @@ func (p *csiProvisioner) Provision(options controller.ProvisionOptions) (*v1.Per
 		return nil, capErr
 	}
 
+	if options.PVC.Spec.DataSource != nil && (rc.clone || rc.snapshot) {
+		contentSource := rep.GetVolume().ContentSource
```

Comment:
Do we store contentSource anywhere in the PV object?

Reply:
Only DataSource is not nil, and the pvc create from snapshot or clone with other pvc